ethereum/permissioning/src/main/java/tech/pegasys/pantheon/ethereum/permissioning/TomlConfigFileParser.java

```diff
@@ -42,8 +42,8 @@ public static TomlParseResult loadConfiguration(final String toml) throws Except
 
     if (result.hasErrors()) {
       final String errors =
-          result.errors().stream().map(TomlParseError::toString).collect(Collectors.joining("%n"));
-      throw new Exception("Invalid TOML configuration : " + errors);
+          result.errors().stream().map(TomlParseError::toString).collect(Collectors.joining("\n"));
+      throw new Exception("Invalid TOML configuration : \n" + errors);
```

Comment:
spurious space between configuration and :

Reply:
removed